lib/src/main/java/org/exoplatform/social/addons/updater/RDBMSMigrationManager.java

```diff
@@ -140,6 +156,10 @@ public void run() {
                       && !MigrationContext.isIdentityDone()) {
                 getIdentityMigrationService().start();
                 updateSettingValue(MigrationContext.SOC_RDBMS_IDENTITY_MIGRATION_KEY, MigrationContext.isIdentityDone());
+
+                if (useMigrationIdentityStorage && MigrationContext.isIdentityDone()) {
+                  identityManager.setIdentityStorage(identityStorage);
```

Comment:
That's what I want because if not, the failed identities will not be migrate again when we restart PLF because they will be recreate (with data retrieve from organization service) when user starting to use PLF, so user's data may be lost.